clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumerImpl.java

```diff
@@ -219,7 +220,7 @@ public CompletableFuture<Void> internalSubscribe(Set<Integer> partitions, PubSub
       }
 
       synchronized (pubSubConsumer) {
```

Comment:
`getTopicAssignment` now locks on pubSubConsumer, we don need this synchronized anymore?

Reply:
We do if only because we later in this synchronized block change the assignment via subscription.